eng/tools/lint-diff/src/runChecks.ts

```diff
@@ -59,30 +59,30 @@ export async function runChecks(
 
       let lintDiffResult: AutorestRunResult;
       try {
-        const executionResult = await execNpmExec(
-          autorestArgs, 
-          { maxBuffer: MAX_EXEC_BUFFER, logger: new ConsoleLogger(true) }
-        );
+        const executionResult = await execNpmExec(autorestArgs, {
+          maxBuffer: MAX_EXEC_BUFFER,
+          logger: new ConsoleLogger(true),
```

Comment:
This will need reconciling with changes from https://statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html